cluster/gce/trusty/configure.sh

```diff
@@ -193,17 +193,19 @@ install_kube_binary_config() {
     cp /tmp/kubernetes/server/bin/kube-scheduler.tar /run/kube-docker-files/
     cp -r /tmp/kubernetes/addons /run/kube-docker-files/
   fi
-  # For a testing cluster, we use kubelet, kube-proxy, and kubectl binaries
-  # from the release tarball and place them in /usr/local/bin. For a non-test
-  # cluster, we use the binaries pre-installed in the image, or pull and place
-  # them in /usr/bin if they are not pre-installed.
-  BINARY_PATH="/usr/bin/"
-  if [ "${TEST_CLUSTER:-}" = "true" ]; then
-    BINARY_PATH="/usr/local/bin/"
-  fi
-  if ! which kubelet > /dev/null || ! which kube-proxy > /dev/null || [ "${TEST_CLUSTER:-}" = "true" ]; then
-    cp /tmp/kubernetes/server/bin/kubelet "${BINARY_PATH}"
-    cp /tmp/kubernetes/server/bin/kubectl "${BINARY_PATH}"
+  # We use the binary from the release tarball if they are not preinstalled, or if this is
+  # a test cluster.
+  if ! which kubelet > /dev/null || ! which kubectl > /dev/null; then
+    cp /tmp/kubernetes/server/bin/kubelet /usr/bin
```

Comment:
nit: Can we define variables for these paths instead of specifying them verbatim in multiple lines?